src/BreathOfTheWild/Mods/MasterModeOptions/rules.txt

```diff
@@ -0,0 +1,111 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = "Master Mode Options"
+path = "The Legend of Zelda: Breath of the Wild/Mods/Master Mode Options"
+description = Allows you to pick and choose which of Nintendo's tweaks |are applied to Master Mode (Does nothing in Normal Mode).|A reboot is required to apply changes made while the game |is running. ||Logic and patching by Torphedo and Fleevoid, with lots of |help from Exzap and leoetlino.
+version = 7
+
+[Default]
+$debug:int = 0
+$bit0:int = 0x1
+$bit1:int = 0x0
+$bit3:int = 0x8
+$bit4:int = 0x10
+$bit5:int = 0x20
+$bit6:int = 0x40
+$bit7:int = 0x80
+$bit8:int = 0x100
+
+[Preset]
+name = Enabled (Vanilla)
+category = IsLastPlayHardMode (Does nothing)

```

Comment:
Maybe place the debug mode enabler at the top of all the settings, and then move all the debug options to the bottom. Like how FPS++ does it's simple and advanced settings. Think it looks neater and generally less confusing to have options get moved down if options were to appear at the top.